Chimera/EventCard.swift

```diff
@@ -0,0 +1,46 @@
+//
+//  EventCard.swift
+//  Chimera
+//
+//  Created by Antonella Giugliano on 11/01/23.
+//
+
+import SwiftUI
+
+struct EventCard: View {
+    
+    let image: String
+    let performer: String
+    let date: String
+    let place: String
+    
+    var body: some View {
+        VStack(alignment: .leading){
+            
+            Image(image)
+                .resizable()
+                .scaledToFill()
+                .frame(width: 356, height: 136)
```

Comment:
Try to not use specific width if the card should be full width because it would be specific only for a specific device